docs/ensuring-repro/docker/index.md

```diff
@@ -34,6 +34,16 @@ We will also use Docker images to run the analysis modules in the `OpenScPCA-nf`
 
 Follow the instructions to [Install Docker Desktop on Mac](https://docs.docker.com/desktop/install/mac-install/) on the Docker website.
 
+#### Apple silicon (M1-3) Mac users
+
+After installing Docker, we recommend that Apple silicon Mac users follow these additional steps:
+
+1. Install Rosetta 2 [as described in the "System Requirements" section in the "Mac with Apple Silicon" tab](https://docs.docker.com/desktop/install/mac-install/#system-requirements).
```

Comment:
Since people have to click a tab to get the instructions, maybe just tell them to install directly with `softwareupdate --install-rosetta` in a terminal window.

But if you keep this, note that on the web page it is ""Mac with Apple silicon"